### PR TITLE
Create connectivity build

### PR DESCRIPTION
**Pull request by philipktlin**
Create a new VSTS build for connectivity test deployment.

Tested in VSTS, https://msazure.visualstudio.com/One/_build/results?buildId=27179684

### CODE REVIEW

File: builds/e2e/templates/connectivity-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+parameters:
```
Comment (and-rewsmith) [inline builds/e2e/templates/connectivity-deploy.yaml]: Shouldn't there be a windows version of this?

File: builds/e2e/connectivity.yaml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        parameters:
+          release.label: 'ct$(agent.group)'
+          edgelet.artifact.name: '$(edgelet.artifact.name)'
+          images.artifact.name: '$(images.artifact.name.linux)'
+          container.registry: '$(container.registry)'
+          container.registry.username: '$(edgebuilds-azurecr-io-username)'
+          container.registry.password: '$(edgebuilds-azurecr-io-pwd)'
+          iotHub.connectionString: '$(EdgeConnectivityTestHubConnString)'
+          eventHub.connectionString: '$(EdgeConnectivityEventHubConnString)'
+          loadGen.message.frequency: '$(loadGen.message.frequency.arm32)'
```
Comment (and-rewsmith) [inline builds/e2e/connectivity.yaml]: nit: newline EOF

**Comment by and-rewsmith** [comment]
What is the plan for supporting windows agents for connectivity tests?

File: builds/e2e/connectivity.yaml (hunk 2 above)
Comment (philipktlin) [reply builds/e2e/connectivity.yaml]: I don't think it is a matter, but updated.

**Comment by philipktlin** [comment]
> What is the plan for supporting windows agents for connectivity tests?

I don't know yet; but not this year.

File: builds/e2e/connectivity.yaml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      - template: templates/connectivity-deploy.yaml
+        parameters:
+          release.label: 'ct$(agent.group)'
+          edgelet.artifact.name: '$(edgelet.artifact.name)'
+          images.artifact.name: '$(images.artifact.name.linux)'
+          container.registry: '$(container.registry)'
+          container.registry.username: '$(edgebuilds-azurecr-io-username)'
+          container.registry.password: '$(edgebuilds-azurecr-io-pwd)'
+          iotHub.connectionString: '$(EdgeConnectivityTestHubConnString)'
+          eventHub.connectionString: '$(EdgeConnectivityEventHubConnString)'
```
Comment (and-rewsmith) [inline builds/e2e/connectivity.yaml]: If we will eventually have multiple hubs for testing due to event hub issues, we will eventually need to index this variable name in keyvault.

Comment (philipktlin) [reply builds/e2e/connectivity.yaml]: You are right.  will see.

**Pull request merged**